agir/front/components/allPages/TopBar/UserMenu.js

```diff
@@ -130,6 +134,15 @@ export const UserMenu = (props) => {
           >
             Paramètres
           </Button>
+          <Button
+            as="Link"
+            route={routeSettings}
```

Comment:
La valeur de `route` normalement devrait être un string correspondant à l'id de la route dans routeConfig, par ex. ici tu peux faire `route="notificationSettings"`. Sinon, pour passer un URL tu peux utiliser `href` (pour les liens externes) ou `to` pour les liens de React router.